eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,49 @@
+param (
+  [Parameter(mandatory = $true)]
+  $pathToArtifacts,
+  [Parameter(mandatory = $true)]
+  $originalDistTags,
+  [Parameter(mandatory = $true)]
+  $intendedTag,
+  [Parameter(mandatory = $true)]
+  $intendedTagVersion
+)
+
+# TODO: delete testing comments below
+#'@azure/eventhubs-checkpointstore-table' alpha, no beta, no GA
+#'@azure/openai' alpha, beta, no GA
+#'@azure/video-indexer-widgets' no alpha, no beta, GA
+#'@azure/template' alpha, beta, GA
+#$pkgProps.PackageId
+
+$ErrorActionPreference = 'Stop'
+$PSNativeCommandUseErrorActionPreference = $true
+
+$tarFile = (Get-ChildItem -Path "$pathToArtifacts/*.tgz")?.Name
+$tempDir = "temp_decompress"
+New-Item -ItemType Directory -Force -Path $tempDir
+tar -xzf $tarFile -C $tempDir
+$pkg = Get-Content -Raw "$tempDir\package\package.json" | ConvertFrom-Json
+$packageName = $pkg.Name
+Remove-Item -Force -Recurse $tempDir
+
+Write-Host "Verify npm tag versions for package $packageName"
+
+$parsedOriginalDistTags = $originalDistTags | ConvertFrom-Json
+
+$npmPkgProp = npm view $packageName --json | ConvertFrom-Json
+$packageDistTags = $npmPkgProp."dist-tags"
+Write-Host "Current dist-tag: $packageDistTags"
+
+if ($packageDistTags."$intendedTag" -ne $intendedTagVersion) {
+  Write-Host "Tag not correctly set, current $intendedTag tag is version $packageDistTags.'$intendedTag' instead of $intendedTagVersion."
+  $correctDistTags = $parsedOriginalDistTags
+  $correctDistTags."$intendedTag" = $intendedTagVersion
+  foreach($tag in $correctDistTags.PSObject.Properties) {
```

Comment:
Did you intend to reset all the original tags here? Or did you only want to set the one tag?

Reply:
intended to set everything back to original, aside from the intended tag change.
This is to prevent situations where the new version is set incorrectly, rather than unset. for example, if the new version is marked as latest instead of dev, latest should be reverted back while dev should be updated